src/TimeoutMigrationTool.NHibernate.OracleAcceptanceTests/TimeoutMigrationTool.NHibernate.OracleAcceptanceTests.csproj

```diff
@@ -10,12 +10,15 @@
     <Compile Include="..\TimeoutMigrationTool.NHibernate.AcceptanceTests\ConfigureExtensions.cs" Link="ConfigureExtensions.cs" />
     <Compile Include="..\TimeoutMigrationTool.NHibernate.AcceptanceTests\EndpointCustomizationConfigurationExtensions.cs" Link="EndpointCustomizationConfigurationExtensions.cs" />
     <Compile Include="..\TimeoutMigrationTool.NHibernate.AcceptanceTests\NHibernateTimeoutsSourceTests.cs" Link="NHibernateTimeoutsSourceTests.cs" />
+    <Compile Include="..\TimeoutMigrationTool.NHibernate.AcceptanceTests\NHibernateToASQEndToEnd.cs" Link="NHibernateToASQEndToEnd.cs" />
     <Compile Include="..\TimeoutMigrationTool.NHibernate.AcceptanceTests\NHibernateToRabbitMqEndToEnd.cs" Link="NHibernateToRabbitMqEndToEnd.cs" />
     <Compile Include="..\TimeoutMigrationTool.NHibernate.AcceptanceTests\DefaultServer.cs" Link="DefaultServer.cs" />
     <Compile Include="..\TimeoutMigrationTool.NHibernate.AcceptanceTests\TestLoggingAdapter.cs" Link="TestLoggingAdapter.cs" />
   </ItemGroup>
 
     <ItemGroup>
+      <PackageReference Include="NServiceBus.Newtonsoft.Json" Version="2.2.0" />
```

Comment:
Here as well

Reply:
I'm not sure I understand. Which third party SDK is missing?